auraed/src/runtime/cells/cell.rs

```diff
@@ -28,19 +28,62 @@
  *                                                                            *
 \* -------------------------------------------------------------------------- */
 
-use crate::runtime::cells::executable::ExecutableError;
-use crate::runtime::cells::{
-    validation::ValidatedCell, CellName, Executable, ExecutableName,
+use super::{
+    validation::ValidatedCell, CellName, Executable, ExecutableError,
+    ExecutableName,
 };
 use cgroups_rs::cgroup_builder::CgroupBuilder;
 use cgroups_rs::{hierarchies, Cgroup, Hierarchy};
-use log::{error, info};
+use log::error;
+use log::info;
 use std::collections::HashMap;
 use std::process::{Command, ExitStatus};
 use thiserror::Error;
 use tonic::Status;
 
-type Result<T> = std::result::Result<T, CellError>;
+pub(crate) type Result<T> = std::result::Result<T, CellError>;
```

Comment:
i prefer to keep this as a separate file, only because it keeps this file focused on the Cell functionality.  you have it at `pub(crate)` visibility anyway so i don't think it matters?

Reply:
Assuming you mean for the `Result` and `CellError`:

I would argue that `CellError` and therefore also `Result` is part of the cell functionality as it represents what could go wrong when using `Cell`, and is likely to change with changes to `Cell`. Maybe I should put `CellError` back at the bottom?

Reply:
I moved the error types to the bottom of the files. See what you think

Reply:
i think i'm stuck on "cells" or "the cell service" should have a single error type and that means it shouldn't be in this file.  however, if you separate the "cell service error" from the "cell error" then i can see why you'd keep the CellError here.

Reply:
There is an argument to be made that `executables` can be its own module, then `cells` would depend on it, and `CellService` would depend on `cells`, but I think breaking them up would be overkill at this point. However, it does help think about why there are multiple error types.

Definitely interested in your thoughts on the below comment too (when you have a chance).

Reply:
but exactly: we wouldn't break them up into their own modules. which is why we shouldn't separate the errors either :)